.github/workflows/resources/unit-test-conf.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: host-volume
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/home/runner/work/jasminegraph/jasminegraph/coverage"
```

Comment:
Why term `jasminegraph` repeated here ? `/home/runner/work/jasminegraph/jasminegraph/coverage`

Reply:
Repository content is found in this path when checkout action is used
![image](https://github.com/miyurud/jasminegraph/assets/76834377/9d17cb21-35e0-4da1-b132-15bd84eddb49)

Reply:
I see, Ok